Rocket.Chat/API/Clients/CommandsClient.swift

```diff
@@ -0,0 +1,31 @@
+//
+//  CommandsClient.swift
+//  Rocket.Chat
+//
+//  Created by Matheus Cardoso on 11/27/17.
+//  Copyright © 2017 Rocket.Chat. All rights reserved.
+//
+
+import RealmSwift
+
+struct CommandsClient: APIClient {
+    let api: AnyAPIFetcher
+
+    func fetchCommands(realm: Realm? = Realm.shared) {
+        api.fetch(CommandsRequest(), succeeded: { result in
+            DispatchQueue.main.async {
+                result.commands?.forEach { command in
+                    try? realm?.write {
+                        realm?.add(command, update: true)
+                    }
+                }
+            }
```

Comment:
What do you think about moving this code (that handles DB operations) to a separated class? 🤔